recipes/lineedit/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "lineedit" %}
+{% set version = "0.1.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lineedit-{{ version }}.tar.gz
+  sha256: f4795479154e350ad9cf6e8be6ad3a78f10f892b99675af2f8140f2d847f6f3f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - pygments
+    - python >=2.7
+    - six >=1.9.0
+    - wcwidth
+
+test:
+  imports:
+    - lineedit
+    - lineedit.deps
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/randy3k/lineedit
+  summary: An readline library based on prompt_toolkit which supports multiple modes
```

Comment:
Huh would have assumed that this means this depends on `prompt_toolkit`, but [that is not the case]( https://github.com/randy3k/lineedit/blob/4262e9852aa50b09ff3f0569e1840b003ed00584/setup.py#L33-L37 )

Reply:
Right, instead an older version of prompt_toolkit is directly vendored, hence the need for 2 license files below.